ui/pages/confirmations/components/confirm/header/header-info.tsx

```diff
@@ -63,7 +63,7 @@ const HeaderInfo = () => {
       properties: {
         action: 'Confirm Screen',
         location: MetaMetricsEventLocation.SignatureConfirmation,
-        signature_type: currentConfirmation?.type,
+        signature_type: currentConfirmation?.msgParams?.signatureMethod,
```

Comment:
`currentConfirmation?.msgParams?.signatureMethod ?? currentConfirmation?.type` will be better as `currentConfirmation?.msgParams?.signatureMethod` is optional fields and may be absent in some signature types.

Reply:
done